board/opentrons/ot2/kernel-patches/0008-build-rv3028-driver-into-kernel.patch

```diff
@@ -0,0 +1,24 @@
+From e4566fa0008b384aba64619399f04ec88f2f840e Mon Sep 17 00:00:00 2001
+From: sanni-t <sanniti@opentrons.com>
+Date: Tue, 23 Mar 2021 18:22:34 +0000
+Subject: [PATCH 8/8] build rv3028 driver into kernel
+
+---
+ arch/arm/configs/bcm2709_defconfig | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
+index 99e08142ed74..32dc7629972c 100644
+--- a/arch/arm/configs/bcm2709_defconfig
++++ b/arch/arm/configs/bcm2709_defconfig
+@@ -1146,6 +1146,7 @@ CONFIG_RTC_DRV_PCF2123=m
+ CONFIG_RTC_DRV_DS3232=m
+ CONFIG_RTC_DRV_PCF2127=m
+ CONFIG_RTC_DRV_RV3029C2=m
++CONFIG_RTC_DRV_RV3028=y
+ CONFIG_DMADEVICES=y
+ CONFIG_DMA_BCM2835=y
+ CONFIG_DMA_BCM2708=y
+-- 
```

Comment:
Modifying kernel configuration through a patch file feels indirect to me. Like it competes with buildroot's own build system.

By comparison, here's what Orange Pi does:

https://github.com/Opentrons/buildroot/blob/4f575ca1e3c0449666229871be8897206cda14d2/configs/orangepi_lite_defconfig#L11

Would that make more sense? I have a hard time following Buildroot's build system, so maybe I'm missing something.

Reply:
Yes, that would work and is probably a better way to do it. Really the outright correct thing to do is make our own kernel fork and a dedicated kernel config, but the fragments approach is fine too.

Reply:
Testing this change